### PR TITLE
Scramble highlighting on alternated lines, only for certain puzzles (4-7 and megaminx) - improves readability

### PR DESCRIPTION
**Pull request by pedrosino**
This fixes https://github.com/cubing/tnoodle/issues/155.

### CODE REVIEW

**Comment by jfly** [comment]
Does this compile for you? Travis is failing with https://travis-ci.org/cubing/tnoodle/builds/51882693#L1238.

Is Arial a monospaced font?

File: webscrambles/src/net/gnehzr/tnoodle/server/webscrambles/ScrambleRequest.java [hunk 1]
```diff
@@ -89,6 +88,7 @@
     private static final int SCRAMBLE_IMAGE_PADDING = 2;
     private static final float MAX_SCRAMBLE_FONT_SIZE = 20;
     private static final float MINIMUM_ONE_LINE_FONT_SIZE = 12;
+    private static final int MIN_LINES_TO_ALTERNATE_HIGHLIGHTING = 5;
```
Comment (jfly) [inline webscrambles/src/net/gnehzr/tnoodle/server/webscrambles/ScrambleRequest.java]: Do we still use this?

**Comment by pedrosino** [comment]
Fixed now.

File: webscrambles/src/net/gnehzr/tnoodle/server/webscrambles/ScrambleRequest.java [hunk 2]
```diff
@@ -749,30 +792,40 @@ private static PdfPTable createTable(PdfWriter docWriter, Document doc, float si
             l.log(Level.INFO, "", e);
         }
 
+        //check if this puzzle should have highlighted scrambles or not
```
Comment (jfly) [inline webscrambles/src/net/gnehzr/tnoodle/server/webscrambles/ScrambleRequest.java]: I don't think we need this comment, the method name is quite clear =)

File: webscrambles/src/net/gnehzr/tnoodle/server/webscrambles/ScrambleRequest.java [hunk 3]
```diff
@@ -749,30 +792,40 @@ private static PdfPTable createTable(PdfWriter docWriter, Document doc, float si
...
-                scrambleChunk.setSplitCharacter(SPLIT_ON_NEWLINES);
-            } else {
-                scrambleChunk.setSplitCharacter(SPLIT_ON_SPACES);
+            Phrase scramblePhrase = new Phrase();
+            boolean oddLine = false;
+            LinkedList<Chunk> lineChunks = splitScrambleToLineChunks(paddedScramble, scrambleFont, availableScrambleWidth);
+            for(Chunk lineChunk : lineChunks) {
+                oddLine = !oddLine;
+                if(highlight && oddLine) {
+                    lineChunk.setBackground(new BaseColor(220, 220, 250));
```
Comment (jfly) [inline webscrambles/src/net/gnehzr/tnoodle/server/webscrambles/ScrambleRequest.java]: Lets pull this color out into a `private static final` variable at the top of this file.

**Comment by pedrosino** [comment]
Done.

**Comment by jfly** [comment]
LGTM! :ship: it!

**Pull request merged**